setting_up_firely_server/deployment/reverseproxy/reverseProxy.rst

```diff
@@ -67,3 +69,6 @@ When using this header, make sure that the header value only contains the domain
 
 - fire/
 - https://fire.ly
+
+Additionally to the ```X-Forwarded-Host`` header, Firely Server will interpret the ```X-Forwarded-Prefix`` header. Based on the header a :ref:`PathBase<hosting_options>` can be dynamically added to each request.
```

Comment:
If X-Forwarded-Prefix is used in the request, is the single static PathBase still evaluated?

Reply:
Yes, in these cases it’s combined. This leads to more confusion I think. I wouldn’t document it, I think.

Reply:
It was the first question that sprung to my mind, so maybe just document that we do not advise using them together. 
Approved nonetheless, this can be done later as well.